src/Functional/InvokeFirst.php

```diff
@@ -11,17 +11,17 @@
 namespace Functional;
 
 use Functional\Exceptions\InvalidArgumentException;
-use Traversable;
 
 /**
  * Calls the method named by $methodName on first object in the collection containing a callable method named
  * $methodName. Any extra arguments passed to invoke will be forwarded on to the method invocation.
  *
- * @param Traversable|array $collection
+ * @param iterable<object> $collection
  * @param string $methodName
```

Comment:
```suggestion
 * @param non-empty-string $methodName
```

might work in several functions

Reply:
Good idea yes :+1: